src/device.h

```diff
@@ -8,13 +8,22 @@
 #define MG_DEVICE_NONE 0        // Dummy system
 #define MG_DEVICE_STM32H5 1     // STM32 H5
 #define MG_DEVICE_STM32H7 2     // STM32 H7
+#define MG_DEVICE_RT1020 3      // IMXRT1020
+#define MG_DEVICE_RT1060 4      // IMXRT1060
 #define MG_DEVICE_CH32V307 100  // WCH CH32V307
 #define MG_DEVICE_CUSTOM 1000   // Custom implementation
 
 #ifndef MG_DEVICE
 #define MG_DEVICE MG_DEVICE_NONE
 #endif
 
+#if MG_DEVICE == MG_DEVICE_RT1020 || MG_DEVICE == MG_DEVICE_RT1060
+#define MG_FLASH_OFFSET 0x60000000  // Offset to memory mapped flash
+#define MG_FLASH_CODE_OFFSET 0x2000 // Offset to code from start of flash
```

Comment:
Why not use mg_flash_start() to handle this ? (same in device_imxrt.c)

Reply:
Ditto. mg_flash_* API was added to fully describe flash layout. We must not have this extra stuff.